### ISSUE TITLE
Privacy Badger blocks Osano's Cookie Consent Panel

### ISSUE THREAD

**Issue by kpawloski** [EFForg/privacybadger#2823]
#### What is your browser and browser version?
Chrome - Version 96.0.4664.110 (Official Build) (64-bit)

#### What is broken and where?
Privacy Badger is blocking the Cookie Preference Center offered by Osano which various sites use to manage cookie preference and honor GPC signals. Osano is a competitor to OneTrust.

#### What is the "culprit" domain?
cmp.osano.com

Sites using Osano:
https://www.mercurynews.com/
https://www.goodrx.com/
https://www.leagueoflegends.com/

Please follow the debug instructions to identify which domain breaks stuff when blocked:
**** ACTION_MAP for osano.com
osano.com {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 0
}
cmp.osano.com {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1640121515624
}
**** SNITCH_MAP for osano.com
osano.com [
  "dailybreeze.com",
  "leagueoflegends.com",
  "mailgun.com"
]

**Comment by ghostwords** [comment]
Hello and thank you for getting in touch!

Does moving the slider for all `osano.com` subdomains to "block cookies" (middle, yellow) position and reloading work around the problem?

**Comment by kpawloski** [comment]
> Hello and thank you for getting in touch!
> 
> Does moving the slider for all `osano.com` subdomains to "block cookies" (middle, yellow) position and reloading the page work around the problem?

Yes, manually flagging it as yellow fixes the problem. You don't need to allow the entire domain and just use cmp.osano.com.

**Comment by ghostwords** [comment]
OK, thanks again! I just added `cmp.osano.com` to the yellowlist. Let me know if there are any further issues.

**Comment by kpawloski** [comment]
> 

Thanks! How long does this change take to propagate out to existing installs? Verified new installs look good.

**Comment by ghostwords** [comment]
About a day or whenever the extension (or the browser) restarts next, whichever comes first.
